### ISSUE TITLE
class heritage L is not an object or null

### ISSUE THREAD

**Issue by saoirse-iontach** [zen-fs/dom#1]
When loading browser.min.js
It result in an Uncaught TypeError: class heritage L is not an object or null 
at  browser.min.js:3:14236 / SyncStore.js:83:7

It seems that class is defined after first reference.
So BrowserFS_DOM is not defined and lib is unusable.

### Workaround:
With `npm  --save-dev esbuild-plugin-external-global`
Update `scripts/build.mjs` as following:
```
import {externalGlobalPlugin} from "esbuild-plugin-external-global";

function externalGlobals(pkg, global, mods) {
  const entries = mods.map(m => [`${pkg}/${m}.js`, global]);
  return Object.fromEntries(entries);
}

const core_externals = externalGlobals('@browserfs/core', 'BrowserFS', [
  'ApiError', 'FileIndex', 'backends/AsyncStore', 'backends/SyncStore',
  'cred', 'file', 'filesystem', 'inode', 'mutex', 'stats', 'utils'
]);

const ctx = await context({
	entryPoints: ['src/index.ts'],
	globalName: 'BrowserFS_DOM',
        /* ... */
        plugins: [
          externalGlobalPlugin(externalGlobals),
          { name: 'watcher', setup(build) {
              /*...*/ 
          }}
	],
});
```

**Comment by dr-vortex** [comment]
Could you please try with the latest release to see if the error has been resolved? (v0.0.6 at the time of writing)

**Comment by saoirse-iontach** [comment]
> Could you please try with the latest release to see if the error has been resolved? (v0.0.6 at the time of writing)

still error:
```
Uncaught TypeError: class heritage A is not an object or null (browser.min.js:3:13279)
    BrowserFS_DOM https://www.unpkg.com/@browserfs/fs-dom/dist/browser.min.js:3
    <anonymous> https://www.unpkg.com/@browserfs/fs-dom/dist/browser.min.js:3
```
```
Uncaught TypeError: class heritage A is not an object or null
    BrowserFS_DOM SyncStore.js:83
    <anonymous> Worker.ts:104
[SyncStore.js:83:7](https://www.unpkg.com/@browserfs/fs-dom@0.0.6/node_modules/@browserfs/core/dist/backends/SyncStore.js)
```

**Comment by dr-vortex** [comment]
Fixed this by bundling not bundling with the core (not the same way as the PR). Thank you very much for spotting this issue @saoirse-iontach!
